### PR TITLE
Merge create_comment use case into main

### PR DESCRIPTION
**Pull request by YathusanK**
This branch contains the implementation of the create comment use case with a data access object that implements this use case's interface. This branch also contains a suite of unit tests for the view and integration of CA layers. Must implement prepareSuccessView in the presenter after the creation of the PostView and PostViewModel.

### CODE REVIEW

File: src/main/java/use_case/create_comment/application_business_rules/CreateCommentInteractor.java
Comment (Pranjal-R-Agrawal) [inline src/main/java/use_case/create_comment/application_business_rules/CreateCommentInteractor.java:1]: The checks in line 23 and 25 could be made stronger by checking if the strings consist only of whitespace characters. Possibly by using trim() and checking if the resultant string equals an empty string.

File: src/test/java/view/CreateCommentViewTest.java
Comment (Pranjal-R-Agrawal) [inline src/test/java/view/CreateCommentViewTest.java:1]: Consider using equals() instead of contains() when checking the message.

**Comment by Pranjal-R-Agrawal** [comment]
Add a TODO to MongoDBDataAccessObjectTest for adding tests for addComment() once the function for fetching comments has been implemented.

**Comment by YathusanK** [comment]
Fixed issues and added recommendations brought up by Pranjal

**Review by Pranjal-R-Agrawal — APPROVED** [review]
The recommended changes have been made.
The commit messages are descriptive.
The test suite is comprehensive, with tests of both the view and the interactor.

**Pull request merged**